compiler/onecc-docker/onecc-docker.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+from distutils.version import StrictVersion
+import json
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
```

Comment:
You can use the function `_run` from `utils` library.
https://github.com/Samsung/ONE/blob/330f9a37cc520c4a5afc91858a8cc2276bb7559a/compiler/one-cmds/utils.py#L204-L211

Reply:
This is invalid answers because `onecc-docker` doesn't have a dependency on `onecc`.